plugins/chat/spec/requests/direct_messages_controller_spec.rb

```diff
@@ -75,7 +75,7 @@ def create_dm_channel(user_ids)
     shared_examples "creating dms" do
       it "creates a new dm channel with username(s) provided" do
         expect {
-          post "/chat/direct_messages/create.json", params: { usernames: [usernames] }
+          post "/chat/api/direct-message-channels.json", params: { target_usernames: [usernames] }
```

Comment:
I updated this spec so we had some specs for the new controller (not perfect but at least everything is still green).